soci/gzip_zinfo.go

```diff
@@ -28,6 +28,24 @@ import (
 	"unsafe"
 )
 
+const windowSize = 32768
+
+// GzipIndexPoint corresponds to gzip_index_point in indexer.c
+type GzipCheckPoint struct {
+	Out    int64  /* corresponding offset in uncompressed data */
+	In     int64  /* offset in input file of first full byte */
+	Bits   int8   /* number of bits (1-7) from byte at in - 1, or 0 */
+	Window []byte /* preceding 32K of uncompressed data */
+}
+
+// GzIndex corresponds to gzip_index in indexer.c
+type CZinfo struct {
```

Comment:
What does the `C` stand for here?

Reply:
This stands for go representation of C struct.

Reply:
Would just keep it as `ZInfo`. `CZinfo` is just bound to confuse people.

Reply:
I think I get the blame for [coining](https://github.com/awslabs/soci-snapshotter/blob/main/soci/gzip_zinfo.go#L32) the term. But, I agree that making it public (capital "C") is leaking an implementation detail.

Reply:
> Does this change mean it is now possible to use the `soci` package as a separate module?

Yes.

Reply:
Removed the test-only code.